### DISCUSSION TITLE
RCP losing its dataset after a while

### DISCUSSION THREAD

**Discussion by dakhnod** [openthread/openthread#10767] [Q&A]
Please correct me if I am wrong, but here is my understanding so far;

When I run `joiner start xxxxxx` and get a `Join successful` in ot-ctl, the dataset is automatically persisted on the nRF52840.
After re-plugging the RCP, the configuration is applied automatically.

Assuming this, why could my RCPs lose their dataset after a few days and refuse to connect to the network?

**Comment by abtink** [comment]
To clarify, in an RCP architecture, the Thread Operational Dataset is not saved on the RCP. It is stored on the host side where the Thread stack support (OT stack) is implemented and running. The RCP functions only as a radio.

**Reply by dakhnod** [reply]
Okay, sounds reasonable. I have two identical chips running the RCP firmware, connected via a FTDI chip to USB.
When I connected the first, I had to join the network. Reasonable, my Host didn't know the network credentials yet.

However, when I connected the second RCP instead and ran `ot-ctl dataset active`, it returned `not found`. No network connection was established.

1. So, is there a persistent state on the RCP after all?

2. What does the `factoryreset` command do?
3. Where in the FS does otbr-posix store the credentials?

Thank you for your help and time!

**Comment by dakhnod** [comment]
I moved this issue to [ot-br-posix](https://github.com/openthread/ot-br-posix/issues/2566), since the issue seems to be related to the host implementation. Thank you.
